### ISSUE TITLE
Popovers can be edited in worker view before team is started

### ISSUE THREAD

**Issue by dretelny** [StanfordHCI/flash-teams#187]
Not sure if this is necessarily a bad thing but worth discussing. See attached screenshot

![image](https://cloud.githubusercontent.com/assets/5275384/3374149/d60e21fe-fbba-11e3-8128-cc3144b44121.png)

**Comment by negarrahmati** [comment]
When I was hiring experts, sending the link to the project on Foundry before hiring the experts usually helped them understand the project and how they were going to work with others. That is why I suggested having read only pop overs for users before starting the team.

**Comment by dretelny** [comment]
I agree that they should be able to see the workflow and that the popovers should be read only. The way it works right now, however, is that the popovers can be edited by the workers before the team has started (e.g., they are not read only) 

Daniela Retelny
Stanford University
Department of Management Science and Engineering
(914) 450-2033
dretelny@stanford.edu
www.danielaretelny.com

**Comment by negarrahmati** [comment]
You are saying that you agree that this is a bug and should be fixed?

Negar

**Comment by dretelny** [comment]
Yes, this is a bug and should be fixed. The popovers should always be read only in the worker view regardless of whether the project has been started. 

Having said that, if we decide to let worker’s author their own workflows then we would want the popovers to be in edit mode...

Daniela Retelny
Stanford University
Department of Management Science and Engineering
(914) 450-2033
dretelny@stanford.edu
www.danielaretelny.com
